anta/tests/snmp.py

```diff
@@ -339,3 +342,117 @@ def test(self) -> None:
             self.result.is_success()
         else:
             self.result.is_failure(f"The following SNMP error counters are not found or have non-zero error counters:\n{error_counters_not_ok}")
+
+
+class VerifySNMPNotificationHost(AntaTest):
+    """Verifies the SNMP notification host (SNMP manager) configurations.
+
+    This test performs the following checks for each specified host:
+
+     1. Verifies that the SNMP host and hostname is configured on the device.
+     2. Verifies that the notification type matches the expected value.
```

Comment:
lets add the following for clarity's sake
```suggestion
     2. Verifies that the notification type ("trap" or "inform") matches the expected value.
```